components/brave_wallet_ui/page/screens/send/send-page/send-screen.tsx

```diff
@@ -39,6 +39,7 @@ export const SendScreen = () => {
         setSelectedSendOption={setSelectedSendOption}
         showSelectTokenModal={showSelectTokenModal}
         selectTokenModalRef={selectTokenModalRef}
+        isAndroid={isAndroid}
```

Comment:
We still need add `isAndroid` as an argument to the `send-screen.tsx` component on line `17`

```
interface Props {
  isAndroid?: boolean
}

export const SendScreen = ({ isAndroid }: Props) => { 
```